backend/app/services/collections/create_collection.py

```
@@ -186,113 +158,311 @@ def _handle_job_failure(
         )
 
 
-def execute_job(
+def execute_setup_job(
     request: dict,
-    with_assistant: bool,
     project_id: int,
     organization_id: int,
     task_id: str,
     job_id: str,
     task_instance,
```

Ruff (0.15.12)

[warning] Unused function argument: `task_instance`

(ARG001)